DH_Vehicles/Classes/DH_IS2CannonPawn.uc

```diff
@@ -11,7 +11,7 @@ defaultproperties
     DriverPositions(0)=(ViewLocation=(X=115.0,Y=-20.0,Z=5.0),ViewFOV=22.5,PositionMesh=SkeletalMesh'DH_IS2_anm.IS2-turret_int',ViewPitchUpLimit=6000,ViewPitchDownLimit=64500,ViewPositiveYawLimit=19000,ViewNegativeYawLimit=-20000,bDrawOverlays=true)
     DriverPositions(1)=(ViewLocation=(X=0.0,Y=0.0,Z=14.0),ViewFOV=75.0,PositionMesh=SkeletalMesh'DH_IS2_anm.IS2-turret_ext',DriverTransitionAnim="VIS2_com_idle_close",ViewPitchUpLimit=2731,ViewPitchDownLimit=64080,ViewPositiveYawLimit=65536,ViewNegativeYawLimit=-65536,bDrawOverlays=true)
     //TO-DO: add some animations to transition from periscope view
-    DriverPositions(2)=(ViewFOV=75.0,PositionMesh=SkeletalMesh'DH_IS2_anm.IS2-turret_int',DriverTransitionAnim="VIS2_com_close",TransitionUpAnim="com_open",ViewPitchUpLimit=4000,ViewPitchDownLimit=64500,ViewPositiveYawLimit=65536,ViewNegativeYawLimit=-65536)
+    DriverPositions(2)=(ViewFOV=72.0,PositionMesh=SkeletalMesh'DH_IS2_anm.IS2-turret_int',DriverTransitionAnim="VIS2_com_close",TransitionUpAnim="com_open",ViewPitchUpLimit=4000,ViewPitchDownLimit=64500,ViewPositiveYawLimit=65536,ViewNegativeYawLimit=-65536)
```

Comment:
In this case, just removing the `ViewFOV` variable here should fall back to the default, so there's no need to re-specify it. Same goes for all the other instances below.